applications/launchpad/gui-react/src/components/Callout/index.tsx

```diff
@@ -28,7 +30,11 @@ const Callout = ({
   }
 
   return (
-    <StyledCallout $type={type} $inverted={inverted}>
+    <StyledCallout
+      $type={type}
+      $inverted={inverted}
+      style={{ background: theme.calloutBackground }}
```

Comment:
`StyledCallout` itself has the CSS rule for the background. The above `style={{ background: ... }}` will overwrite this.